persistence/src/test/java/edu/unc/lib/dl/update/AccessControlUIPFilterTest.java

```diff
@@ -34,8 +38,7 @@
 import edu.unc.lib.dl.util.ContentModelHelper;
 import edu.unc.lib.dl.xml.JDOMNamespaceUtil;
 
-import static org.mockito.Mockito.*;
-
+@Ignore
```

Comment:
is this class just here to avoid compilation errors?

Reply:
Deleting the test likely wouldn't cause any compilation issues, I mainly left it there since the code it tests is still around, in case we need to reference it later when deciding how to replace the code